client/src/reducers/index.js

```diff
@@ -9,6 +9,21 @@ export default (state, action) => {
     case "FETCH_USER_SUCCESS": {
       return { ...state, user: action.user, loading: false, error: null };
     }
+    case "LOGOUT": {
+      return { ...state, loading: true };
+    }
+    case "LOGOUT_USER_ERROR": {
+      return { ...state, error: action.error, loading: false };
+    }
+    case "LOGOUT_USER_SUCCESS": {
+      return { ...state, user: {}, loading: false, error: null };
+    }
+    case "LOGIN": {
+      return { ...state, user: action.user, loading: false, error: null };
+    }
+    case "SIGNUP": {
+      return { ...state, user: action.user, loading: false, error: null };
+    }
```

Comment:
I think there is a similar reducer for logout in there, it is called REMOVE_USER etc. Can remove that if using these reducers.

Reply:
Decided to leave it for now, maybe we will use it the future, if not - will remove it